ta/os_test/ta_entry.c

```diff
@@ -109,6 +109,9 @@ TEE_Result TA_InvokeCommandEntryPoint(void *pSessionContext,
 	case TA_OS_TEST_CMD_PARAMS:
 		return ta_entry_params(nParamTypes, pParams);
 
+	case TA_OS_TEST_CMD_NULL_MEMREF_PARAMS:
+		return ta_entry_params_tmpmem(nParamTypes, pParams);
```

Comment:
for consistency, prefer rename `ta_entry_params_tmpmem()` into `ta_entry_null_memref()`.